Ascend.java

```diff
@@ -3,5 +3,20 @@
 public class Ascend {

```

Comment:
OVERALL:
Good assignment.
1. There are some logical issues in code of time format please review my comments.
2. Names should be better
3. Don't need to split every word into different string
4. See my comment regarding Integer.toString()

More consistent names of commits...